test/ClangImporter/availability_returns_twice.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-frontend -typecheck -verify %s
+
+#if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
+  import Darwin
+  typealias JumpBuffer = Int32
+#else
+  import Glibc
+  typealias JumpBuffer = __jmp_buf_tag
+#endif
+
+func test_unavailable_returns_twice_function() {
+  var x: JumpBuffer
```

Comment:
Unfortunately no, I get the following error on macOS: `cannot convert value of type 'jmp_buf' (aka '(Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32, Int32)') to expected argument type 'Int32'`. It works on Linux, though, so I'll use `jmp_buf` there instead of `__jmp_buf_tag`.